content/docs/1.1.0/snapshots-and-backups/backup-and-restore/set-backup-target.md

```diff
@@ -48,8 +48,7 @@ This page covers the following topics:
 3. Create a Kubernetes secret with a name such as `aws-secret` in the namespace where longhorn is placed(`longhorn-system` by default). For help creating a secret, refer to the [Kubernetes documentation.](https://kubernetes.io/docs/concepts/configuration/secret/) The secret must be created in the `longhorn-system` namespace for Longhorn to access it. Put the following key-value pairs in the secret:
 
```

Comment:
The line above says `put the following key-value pairs in the secret`, which we should update.